build.gradle.kts

```diff
@@ -1,14 +1,20 @@
 ext["grpcVersion"] = "1.47.0"
+ext["guavaVersion"] = "31.0.1-android" // This version is in sync with grpcVersion
```

Comment:
wth?  do we really want to pin to an android version?

Reply:
That is the version used internally by the grpc libraries and the one we were transitively using before. It is the recommended version to use if you want android and JRE compatibility, as per [https://github.com/google/guava/wiki/ReleasePolicy#flavors](url). I don't mind changing it, but not using it could be a problem if someone comes up with a use case for momento directly in an android app.

Reply:
nah just surprising but your explanation makes sense.